packages/core/src/tracing/instrumentation/cloud/aws-sdk/v2/sns.js

```diff
@@ -103,6 +104,12 @@ class InstanaAWSSNS extends InstanaAWSProduct {
       return;
     }
 
+    // SQS has a limit of 10 message attributes, we need to add two attributes.
+    if (Object.keys(attributes).length >= 9) {
+      logTooManyAttributesWarningOnce(attributes, 2);
```

Comment:
*Suggestion*: Pass the logger from the actual instrumentation

e.g. `packages/core/src/tracing/instrumentation/cloud/aws-sdk/v3/sqs.js` is using the logger name `tracing/sqs/v3`

If the warning appears, there will be two different logger names for the same context.